R/group_apply.R

```diff
@@ -22,7 +22,7 @@
 #' value (see examples).
 #'
 #' @return A \code{data.frame} of the outputs from the selected function, with
-#' appended column(s) indicating the user-defined groups. If a single vector
+#' prepended column(s) indicating the user-defined groups. If a single vector
```

Comment:
'Prepended' feels a bit jargon-y, might be nicer to go with e.g. 'added' instead

Reply:
No worries, it's your choice :)